.github/workflows/new-issues-to-triage.yml

```diff
@@ -0,0 +1,52 @@
+name: New Issues to Triage
+
+on:
+    issues:
+        types: [opened]
+
+jobs:
+    new-issue-to-triage:
+        runs-on: ubuntu-latest
+        name: New Issue to Triage
+        # Based on https://docs.github.com/en/issues/trying-out-the-new-projects-experience/automating-projects#example-workflow-authenticating-with-a-personal-access-token
+        steps:
+            - name: Get project data
+              env:
+                  GITHUB_TOKEN: ${{ secrets.TRIAGE_WORKFLOW_TOKEN }}
+                  ORGANIZATION: pulumi
+                  PROJECT_NUMBER: 33
+              run: |
+                  gh api graphql -f query='
+                    query($org: String!, $number: Int!) {
```

Comment:
🤯  this is cool! how'd you determine this query? might be helpful to add a link to the documentation you used to produce this.

Reply:
Agree, if only I had it! I'm just copying this workflow out of another other repo (which was set up by someone who no longer works here).